### ISSUE TITLE
OpenMP dependency

### ISSUE THREAD

**Issue by filiperinaldi** [KIT-MRT/mrt_cmake_modules#6]
cmake/Modules/UseMrtStdCompilerFlags.cmake seems to require **OpenMP**:

```
#add OpenMP
find_package(OpenMP REQUIRED)
set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
```
If  mrt_cmake_modules needs this package, then I assume this dependency should be expressed in its package.xml.

Can any one please clarify it?

**Comment by poggenhans** [comment]
Thanks for reporting. I think this hidden dependency never showed up as a problem, because - at least with GCC - OpenMP is always installed and found. We'll fix that.

**Comment by poggenhans** [comment]
Fixed in 0adfebe58c3210326e52fc54ff14ca1b21ab4a0f by making it optional instead of `REQUIRED`.

Adding it to the package.xml is unfortunately not possible. There is no rosdep definition for openmp. ROS itself seems to handle it similarly by making it optional and not specifying it in their package.xml.
